roles/php/tasks/main.yml

```diff
@@ -45,8 +45,13 @@
   when: php72_status.stat.exists
   notify: reload php-fpm
 
-- name: PHP configuration file
+- name: Copy PHP-FPM configuration file
   template:
-    src: php.ini.j2
+    src: php-fpm.ini.j2
     dest: /etc/php/7.4/fpm/php.ini
   notify: reload php-fpm
+
+- name: Copy PHP CLI configuration file
+  template:
+    src: php-fpm.ini.j2
```

Comment:
Should this be?

```diff
-     src: php-fpm.ini.j2 
+     src: php-cli.ini.j2 
```

Reply:
@TangRufus you're right. I've missed it. Should I submit another PR to fix that? /cc @swalkinshaw ?

Reply:
Yes, please

Reply:
@TangRufus done, https://github.com/roots/trellis/pull/1262